xyz-psql-connector/src/main/java/com/here/xyz/psql/DatabaseHandler.java

```diff
@@ -366,6 +369,23 @@ protected XyzResponse executeModifyFeatures(ModifyFeaturesEvent event) throws Ex
         }
     }
 
+    /**
+     * NOTE: This is a workaround for tables which have no unique constraint
+     * TODO: Remove this workaround once all constraints have been adjusted accordingly
+     */
+    private void checkUniqueTableConstraint(ModifyFeaturesEvent event) throws SQLException {
+        boolean uniqueConstraintExists = new SQLQuery("""
+            SELECT 1 FROM pg_catalog.pg_constraint con INNER JOIN pg_catalog.pg_class rel ON rel.oid = con.conrelid
```

Comment:
just a not, I assume this can be done without a join like:

 select 1 from pg_constraint
 where 1 = 1
   and conname = '94c9b059dee50c412100a299eb5bb477_unique'
   and connamespace::regnamespace::text = 'public'